tg_bot/modules/bans.py

```diff
@@ -191,6 +191,66 @@ def unban(bot: Bot, update: Update, args: List[str]) -> str:
     return log
 
 
+@run_async
+@bot_admin
```

Comment:
All the `return ""` are useless, given this hasn't been set as @loggable. Use normal `return` instead.